### DISCUSSION TITLE
kiali-aes-token cookie attributes

### DISCUSSION THREAD

**Discussion by stanenbaum-va** [kiali/kiali#6907] [Q&A]
We leverage Kiali on our team, and received some WASA scan results that flagged the kiali-aes-token with a low vulnerability.

> The application sets HTTPOnly flag and does not set Secure flag on session cookie that contain sensitive information or controls authentication

Just curious if this was unintentionally left out of the session persistor [code](https://github.com/kiali/kiali/blob/master/business/authentication/session_persistor.go#L135-L141), and if it would be worthy of opening up an issue to change.

Thanks so much!

**Comment by jmazzitelli** [comment]
It is probably not there because Kiali can be served over http if the kiali installer wanted it configured like that. I suppose if this were to be added, the code would need to check if the kiali server endpoint is http or https and set that secure flag accordingly.

**Reply by stanenbaum-va** [reply]
Would creating a bug (or feature, not sure which would be appropriate) for this something that would make sense?

**Reply by jmazzitelli** [reply]
Sure, you can enter an enhancement request in github. I wouldn't call it a bug per-se, but semantics aside, feel free to create a github issue. https://github.com/kiali/kiali/issues/new/choose

**Comment by jmazzitelli** [comment]
PR has been submitted for this:
* https://github.com/kiali/kiali/pull/6930
